evalai/submissions.py

```diff
@@ -1,9 +1,15 @@
 import click
 
-from click import echo
+from evalai.utils.submissions import display_submission_details
 
 
-@click.command()
-def submissions():
-    """Example script."""
-    echo('Hello Submissions!')
+@click.group(invoke_without_command=True)
+@click.argument('SUBMISSION', type=int)
+def submission(submission):
+    """
+    View submission specific details.
+    """
+    """
+    Invoked by `evalai submission SUBMISSION`.
+    """
+    display_submission_details(submission)
```

Comment:
Is it `print_x` everywhere else? Or `display_x`? If it is print, please rename this to ensure consistency.

Reply:
It's `display_x` for fetching the data to be printed, `pretty_print_x` does the printing